### PR TITLE
Fix return parameter to match the implementation

### PR DESCRIPTION
**Pull request by sebastianbenz**
Renamed status -> success

### CODE REVIEW

**Comment by sebastianbenz** [comment]
I signed it!

File: extensions/amp-access/amp-access-spec.md [hunk 1]
```diff
@@ -372,7 +372,7 @@ https://publisher.com/amp-login.html?
 Notice that the “return” URL parameter is added by the AMP Runtime automatically if `RETURN_URL` substitution is not
 specified. Once Login Page completes its work, it must redirect back to the specified “Return URL” with the following format:
 ```
-RETURN_URL#status=true|false
+RETURN_URL#success=true|false
 ```
 Notice the use of a URL hash parameter “status”. The value is either “true” or “false” depending on whether the login succeeds or is abandoned. Ideally the Login Page, when possible, will send the signal in cases of both success or failure.
```
Comment (dvoytenko) [inline extensions/amp-access/amp-access-spec.md]: Please change it here as well: "URL hash parameter "status""

**Comment by dvoytenko** [comment]
Oops. LGTM with one small request. Thanks for spotting!

**Comment by sebastianbenz** [comment]
PTAL

**Comment by dvoytenko** [comment]
Great! Will merge as soon as green.

**Pull request merged**